app/src/main/java/com/bnyro/translate/api/deepl/DeepL.kt

```diff
@@ -37,4 +40,22 @@ interface DeepL {
         @Query("target_lang") target: String,
         @Query("text") query: String
     ): DeeplTranslationResponse
+
+    @Headers(
+        "Accept: */*",
+        "Accept-Language: en-US,en;q=0.5",
+        "Authorization: None",
+        "Content-Type: application/json; charset=utf-8",
+        "Host: www2.deepl.com",
+        "Origin: chrome-extension://cofdbpoegempjloogbagkncekinflcnj",
+        "referer: https://www.deepl.com/",
+        "Sec-Fetch-Dest: empty",
+        "Sec-Fetch-Mode: cors",
+        "Sec-Fetch-Site: none",
+        "User-Agent: DeepLBrowserExtension/1.18.0 Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/124.0.0.0 Safari/537.36"
```

Comment:
Please move the user agent string to a constant, it will probably change in the future and thus we should place it in a different variable for ease of updating.